plugins/hiscore/init.lua

```diff
@@ -95,17 +95,31 @@ function hiscore.startplugin()
 	end
 
 
-	local function read_hiscore_dat ()
-	  local file = io.open( hiscoredata_path, "r" );
-	  local rm_match;
-	  if not file then
-		file = io.open( hiscore_plugin_path .. "/hiscore.dat", "r" );
+	local function read_hiscore_dat (hiscoredata_path)
+	  -- emu.print_verbose( "hiscore_plugin_path: " .. hiscore_plugin_path );
+	  local file = io.open( hiscore_plugin_path .. "/" .. hiscoredata_path, "r" );
+	  if file == nil then
+		-- file not found
+		emu.print_verbose( "hiscore file not found: " .. hiscoredata_path );
+		return ""
 	  end
+	  local rm_match;
+	  local rm_match_crc = 0;
 	  if emu.softname() ~= "" then
 		local soft = emu.softname():match("([^:]*)$")
-		rm_match = '^' .. emu.romname() .. ',' .. soft .. ':';
+		rm_match = emu.romname() .. ',' .. soft .. ':';
```

Comment:
Changing this to not only match the start doesn’t seem like a desirable change.

Reply:
right, i'm fixing this...